lib/ui/LandingPage.dart

```diff
@@ -0,0 +1,240 @@
+    EdgeInsets actionEdgePadding,
+    EdgeInsets footerTextEdgePadding,
+    double subtitleLineSpace,
+    double detailTextLineSpace,
+    double actionLineSpace,
+    int detailTextMaxLines,
+    int footerTextMaxLines,
+  }) {
+    return LandingPageStyle(
+      detailTextStyle: detailTextStyle ?? this.detailTextStyle,
+      footerTextStyle: footerTextStyle ?? this.footerTextStyle,
+      headerEdgePadding: headerEdgePadding ?? this.headerEdgePadding,
+      detailTextEdgePadding:
+      detailTextEdgePadding ?? this.detailTextEdgePadding,
+      actionEdgePadding: actionEdgePadding ?? this.actionEdgePadding,
+      footerTextEdgePadding: footerTextEdgePadding ?? this.footerTextEdgePadding,
+      subtitleLineSpace: subtitleLineSpace ?? this.subtitleLineSpace,
+      detailTextLineSpace: detailTextLineSpace ?? this.detailTextLineSpace,
+      actionLineSpace: actionLineSpace ?? this.actionLineSpace,
+      detailTextMaxLines: detailTextMaxLines ?? this.detailTextMaxLines,
+      footerTextMaxLines: footerTextMaxLines ?? this.footerTextMaxLines,
+    );
+  }
+}
+
+class _DefaultStyle extends LandingPageStyle {
+  final TextStyle detailTextStyle = const TextStyle(
+      fontSize: 17,
+      height: 1.1,
+      fontWeight: FontWeight.normal,
+      color: Color(0xff4c4e6e));
+  final TextStyle footerTextStyle = const TextStyle(
+      fontWeight: FontWeight.normal, fontSize: 15, color: Color(0xff4c4e6e));
+
+  final EdgeInsets headerEdgePadding = const EdgeInsets.only(
+      top: 56, bottom: 20.5);
+  final EdgeInsets detailTextEdgePadding =
+  const EdgeInsets.only(left: 39.75, right: 39.75);
+  final EdgeInsets actionEdgePadding = const EdgeInsets.only(
+      left: 34, right: 34);
+  final EdgeInsets footerTextEdgePadding = const EdgeInsets.only(
+      left: 30, right: 30, bottom: 16);
+
+  final double subtitleLineSpace = 13;
+  final double detailTextLineSpace = 40;
+  final double actionLineSpace = 15;
+  final int detailTextMaxLines = 3;
+  final int footerTextMaxLines = 1;
+
+  const _DefaultStyle({
+    TextStyle detailTextStyle,
+    TextStyle footerTextStyle,
+    EdgeInsets headerEdgePadding,
+    EdgeInsets detailTextEdgePadding,
+    EdgeInsets actionEdgePadding,
+    EdgeInsets footerTextEdgePadding,
+    double subtitleLineSpace,
+    double detailTextLineSpace,
+    double actionLineSpace,
+    int detailTextMaxLines,
+    int footerTextMaxLines,
+  });
+}
+
+const LandingPageStyle _defaultStyle = const _DefaultStyle();
+
+class LandingPage extends StatelessWidget {
+  final LandingPageViewModel _viewModel;
+  final LandingPageStyle _style;
+
+  const LandingPage({Key key,
+    LandingPageViewModel viewModel,
+    LandingPageStyle style = _defaultStyle})
+      : this._viewModel = viewModel,
+        this._style = style,
+        super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      body: SingleChildScrollView(
+        child: Column(
+          children: <Widget>[
```

Comment:
Here I see two main containers the footer and the top. The bottom container that contains the button and the "language selector" and the other container that contains the header, subtitle and the detail text so you can use a `Column` with a `Expanded` and `flex: ` value  to avoid adding a lot of `Padding` on the child widgets

<img width="311" alt="Screenshot 2019-07-16 at 12 46 31" src="https://user-images.githubusercontent.com/23307893/61288601-c2338e80-a7c7-11e9-89a0-535db5c43ad3.png">